src/hooks/use-core.ts

```diff
@@ -1,17 +1,19 @@
-import { useElement, useState, useEffect } from '@nebula.js/stardust';
+import { useElement, useLayout, useState, useEffect  } from '@nebula.js/stardust';
 import useViewState from './use-view-state';
 
 const useCore = () => {
   const element = useElement();
+  const layout = useLayout();
   const viewState = useViewState();
 
   const [core, setCore] = useState();
   useEffect(() => {
     if (!viewState || !element) return undefined;
     setCore({
       element,
+      layout,
     });
-  }, [viewState, element]);
+  }, [viewState, element, layout]);
```

Comment:
I think it makes sense to use the layout-service from qlik-chart-modules instead of the layout directly. It comes with some handy functions.

Reply:
Done